DetectDynamicJS.py

```diff
@@ -70,6 +70,26 @@ def doPassiveScan(self, baseRequestResponse):
         self._requestResponse = baseRequestResponse
         self._helpers = self._callbacks.getHelpers()
         
+        response = self._requestResponse.getResponse()
+        responseInfo = self._helpers.analyzeResponse(response)
+        
+        # Scan only if the statusCode is 200
+        statusCode = responseInfo.getStatusCode()
+        if statusCode != 200:
+            return None
+        
+        # Check if the script is world readable
+        resHeaders = self._helpers.analyzeResponse(response).getHeaders()
```

Comment:
It was for acces-control-allow-origin header, not for the status code. You are right, the status code I do want. Sorry, I thought it was obvious, I'll try to be more exact on what I mean next time.